ui/pages/keychains/reveal-seed.js

```diff
@@ -44,9 +44,7 @@ import { endTrace, trace, TraceName } from '../../../shared/lib/trace';
 const PASSWORD_PROMPT_SCREEN = 'PASSWORD_PROMPT_SCREEN';
 const REVEAL_SEED_SCREEN = 'REVEAL_SEED_SCREEN';
 
-export default function RevealSeedPage() {
-  const history = useHistory();
-  const { keyringId } = useParams();
+function RevealSeedPage({ navigate, keyringId }) {
```

Comment:
Can you explain in short how this works now? From where does the `navigate` reference comes from? And why is it not possible to use `Navigate` from `react-router-dom-v5-compat`? I see that this component is older (probably problematic) and deserves refactoring, but I don't understand the exact issues 😅